### ISSUE TITLE
trial version of virtualbox, nvda is not reading the content and name of the lists 

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#13182]
### Steps to reproduce:
download the trial version of virtualbox, download from the link: 
https://www.virtualbox.org/wiki/Testbuilds#Developmentsnapshots
now i can for example create a virtual machine, i created a virtual machine called windows.
now on the main screen of the virtual box walk with tabe
nvda only advertises lists.
doing the test with the narrator, the narrator, when walking with tabe in a list, informs the windows title, which is the name of the virtual machine I created, and when walking with tabe, the narrator informs of the existence of the other list.
in ambus, screen readers when walking with arrows through the lists, nothing is informed, it seems that the list is empty, but I don't know if they really are, I don't have a vision to fill.
Now in another test, access settings in the files menu.
There is a list of categories where nvda only reads the help text, but when walking with arrows through the list nvda does not announce the contents of this list, although nvda does not announce the contents of the list when walking with arrows the contents of the window alters, which can be seen when browsing with tabe. 
### Actual behavior:
as described so when the list gains focus when walking with tab and nvda does not read its title, and when walking through the list with arrows nvda does not read the contents of the lists. 
### Expected behavior:
when a list gains focus with tabe nvda should announce the title of the list.
when walking with arrows through the lists nvda should announce the contents of the list. 
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
nvda.exe, NVDA alpha-24329,d3444424

#### Windows version:
11 22000.376
#### Name and version of other software in use when reproducing the issue:
VirtualBox.exe, Oracle VM VirtualBox 6.1.97.148976

#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
https://www.virtualbox.org/ticket/629#comment:30
sorry if I'm not on the right track, but as described in the link call the virtual box seems to be undergoing changes when using new versions of qt. 
does nvda also need to be changed to support new versions of qt? 
log:
[vbdev.txt](https://github.com/nvaccess/nvda/files/7747174/vbdev.txt)

**Comment by gregjozk** [comment]
Please describe your findings also on given VB ticket, so VB developers will be aware about a problem.

**Comment by fernando-jose-silva** [comment]
https://www.virtualbox.org/ticket/629#comment:31

follow the link with the update I made on the oracle tick.
I am available.

**Comment by josephsl** [comment]
Hi,

Any updates as of May 2022, or for that matter, did Oracle manage to track this issue and fix it from Virtual Box?

Thanks.

**Comment by fernando-jose-silva** [comment]
in the comment I made on the call to oracle I never received a response

**Comment by josephsl** [comment]
Hi,

I bet this issue still occurs in latest Virtual Box as the download link is for a development build. If so, I advise asking Oracle about this issue again and see if they say this is something to be fixed from the screen reader side.

Thanks.

**Comment by fernando-jose-silva** [comment]
I tried to download the virtual test box from this links but the links for me are not available.
https://www.virtualbox.org/wiki/Testbuilds#Developmentsnapshots

**Comment by josephsl** [comment]
Hi,

Can confirm this - I get a 404 error, in which case I think the next alternative is testing the latest stable version of Virtual Box.

Thanks.

**Comment by fernando-jose-silva** [comment]
We've made great progress here.
On the initial screen, nvda already announces the name of the list and the item in the list, which in this case is the virtual machine created.
the only criticism of nvda in this list is that when navigating with tab and finding a list with 0 items nvda announces list only, narrator announces list with 0 items. Because I think this is important, with nvda I navigate with tabe and I just read the list, I navigate with the arrows and I don't dare anything, because the list is empty, but as a user I can be in doubt if the list is empty or if it's the nvda that does not read the items on the list, when browsing with the narrator this question does not exist, because the narrator already announces 0 items.
However this is not just a problem on this screen, nvda never announces 0 items in any list, which in my view is a flaw, or at least it should be configurable, and I'm sure this is part of old requests here.
In the other test, when browsing the list in settings, nvda also announces the list items.
Finally, because of the evolution of some software in the meantime, the problem was solved.
You can end the call if desired.
Finally, it follows the current software versions.
nvda.exe, NVDA alpha-25324,0f08ccb7

VirtualBox.exe, Oracle VM VirtualBox 6.1.34.150636
windows 11 22000.652

**Comment by josephsl** [comment]
Hi, letting NVDA announce “0 items” is something that could go in a separate list, and NVDA has been behaving like what you’ve described as long as I remember (over ten years). In the meantime I’ll go ahead and close this issue. Thanks.
